.gitignore

```diff
@@ -1,5 +1,20 @@
+.idea
+vendor/
+
+# composer
 composer.lock
 composer.phar
-vendor/
+
+# osx
 .DS_Store
-.idea
+.DS_Store?
+._*
+.Spotlight-V100
+.Trashes
+
+# npm
+.npm/node_modules
+node_modules
+
+# generated files
+templates/theme/css
```

Comment:
What generated files are we talking about here? these seems to belong in the gitignore file in the ```templates/theme``` directory and not here.